From cf28d11eff5d9edbb5330eed2e2bb3e75e4bc066 Mon Sep 17 00:00:00 2001
From: He Sheng <hesheng@wxiat.com>
Date: Fri, 6 May 2022 10:45:56 +0800
Subject: [PATCH] sw64: kapi: remove unimplemented IPLs
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: cf28d11eff5d9edbb5330eed2e2bb3e75e4bc066
Modified-by-SEL: No


Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I56OLG

--------------------------------

There are only two IPLs so far on sw64.

Signed-off-by: He Sheng <hesheng@wxiat.com>

Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/include/asm/irqflags.h | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/arch/sw_64/include/asm/irqflags.h b/arch/sw_64/include/asm/irqflags.h
index 6101b6ad2e99..b4440f25a51d 100644
--- a/arch/sw_64/include/asm/irqflags.h
+++ b/arch/sw_64/include/asm/irqflags.h
@@ -5,14 +5,6 @@
 #include <asm/hmcall.h>
 
 #define IPL_MIN		0
-#define IPL_SW0		1
-#define IPL_SW1		2
-#define IPL_DEV0	3
-#define IPL_DEV1	4
-#define IPL_TIMER	5
-#define IPL_PERF	6
-#define IPL_POWERFAIL	6
-#define IPL_MCHECK	7
 #define IPL_MAX		7
 
 #define getipl()		(rdps() & 7)
-- 
2.33.0

